backend/tests/api/test_booking.py

```diff
@@ -580,3 +610,19 @@ async def test_is_booking_available(
     )
 
     assert await booking._is_booking_available(booking_row) is True
+
+
+@mark.asyncio
+@database.transaction(force_rollback=True)
+async def test_get_limit_for_booking_feature(
+    some_config_rows,
+    some_licenses,
+    insert_objects,
+):
```

Comment:
Docstring is missing

Reply:
Thanks, I added it in the latest commit.